assets/styles/mixins/theme-dark.scss

```
@@ -109,7 +109,7 @@
    --kbin-boosted-color: var(--kbin-upvoted-color);
 
   // alerts
-  --kbin-alert-info-bg: rgba(153,116,4,0.15);
+  --kbin-alert-info-bg: rgba(153,116,4,1);
```

```suggestion
  --kbin-alert-info-bg: rgba(153,116,4,0.9);
```

What about 0.9? Still looks good I think. And just ever so slightly transparent.

---

I am ok with that, but it does not follow suit with the green alert not being transparent?